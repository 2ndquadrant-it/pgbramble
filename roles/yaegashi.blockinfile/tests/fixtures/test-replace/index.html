<html><head><title>111 222 333</title></head><Body><p>!!! @@@ ###</p></body></html>
