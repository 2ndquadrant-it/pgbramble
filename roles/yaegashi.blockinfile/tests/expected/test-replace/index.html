<html><head><title>111 222 333</title></head><Body>
<!-- BEGIN ANSIBLE MANAGED BLOCK -->
<h2>aaa bbb ccc</h2>
<h2>AAA BBB CCC</h2>
<!-- END ANSIBLE MANAGED BLOCK -->
<p>!!! @@@ ###</p></body></html>
